### DISCUSSION TITLE
V2: Connecting to BNB Beacon Chain connection issue

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1554] [Developer Support]
Hello all,

I am trying to integrate with walletconnect 2.0. However, I'm experiencing some issues when it comes to establishing the connection with the BNB Beacon Chain. I'm using this
```

const { uri, approval } = await client.connect({
  requiredNamespaces,
});
```

where the` requiredNamespaces` object looks like this

```
{
    "cosmos": {
        "methods": [
            "cosmos_signDirect",
            "cosmos_signAmino"
        ],
        "chains": [
            "cosmos:Binance-Chain-Tigris"
        ],
        "events": []
    }
}

```

Note: this works for other chains like ETH (mainnet) or THORChain. But with BNB Beacon Chain it will simply reject the session

**Comment by xzilja** [comment]
@GrahamMcBain hard to tell initially, some ideas:

1. Does wallet that you are connecting to support `cosmos:Binance-Chain-Tigris`
2. Is `cosmos:Binance-Chain-Tigris` correct chain id?

Namespace looks correct, so given that eth / thor work there shouldn't be anything special in terms of how sign v2 sdk handles this
